config.py

```diff
@@ -60,7 +60,7 @@ class ProductionConfig(Config):
     @classmethod
     def init_app(cls, app):
         Config.init_app(app)
-
+        assert os.environ.get('SECRET_KEY'), 'SECRET_KEY IS NOT SET!'
```

Comment:
Can you check the SECRET_KEY field of the app's config object instead? That way this code isn't coupled to env based configuration.